### PR TITLE
Update set-adaccountpassword.md

### PR DESCRIPTION
**Pull request by sukihanabi**
#964 
**Action Taken:** 
Removed 
"You must set the *OldPassword* and the *NewPassword* parameters to set the password unless you specify the *Reset* parameter.
When you specify the *Reset* parameter, the password is set to the *NewPassword* value that you provide and the *OldPassword* parameter is not required."

In Example 1: Command will run with / without -Reset , so no added notes.
In Example 2: Added "Using -Newpassword with a value without providing an -Oldpassword parameter value will also reset the password."

**Test the following cmd:**

1.) Run CMd With an -OldPassword , Case sensitive with password requirements
tried several password it does not accept.

Cmd:
Set-ADAccountPassword -Identity Name -OldPassword (ConvertTo-SecureString -AsPlainText "p@ssw0rd" -Force) -NewPassword (ConvertTo-SecureString -AsPlainText "qwert@12345" -Force)

Screenshot:
https://snag.gy/fVInAi.jpg

2.) Run Cmd: Without -OldPassword , Password reset works

Cmd:
Set-ADAccountPassword -Identity Name (ConvertTo-SecureString -AsPlainText "p@ssw0rd" -Force) -NewPassword (ConvertTo-SecureString -AsPlainText "qwert@12345" -Force)

Screenshot: 
https://snag.gy/fzUQDb.jpg

3.) Run CMD: With -Reset and without -Reset , both cmdlet works

Cmd:
Set-ADAccountPassword -Identity 'Name' -Reset -NewPassword (ConvertTo-SecureString -AsPlainText "p@ssw0rd" -Force)

Set-ADAccountPassword -Identity 'Name' -NewPassword (ConvertTo-SecureString -AsPlainText "p@ssw0rd" -Force)

Screenshot:
https://snag.gy/cx8nf4.jpg

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/addsadministration/set-adaccountpassword.md [hunk 1]
```diff
@@ -68,6 +65,7 @@ PS C:\> Set-ADAccountPassword -Identity elisada -OldPassword (ConvertTo-SecureSt
 ```
 
 This command sets the password of the user account with SamAccountName elisada to qwert@12345.
+Using -Newpassword with a value without providing an -Oldpassword parameter value will also reset the password.
```
Comment (JohanFreelancer9) [inline docset/windows/addsadministration/set-adaccountpassword.md]:
```suggestion
Using -NewPassword with a value without providing an -OldPassword parameter value will also reset the password.
```

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by dariomws306** [comment]
Looks good.

**Comment by dariomws306** [comment]
Sorry one comment. I think the same fix should be applied to W2012 and W2012R2 versions.

Thanks!

**Comment by get-itips** [comment]
ok with @dariomws306

**Comment by tseward** [comment]
Agree with @dariomws306

**Comment by e0i** [comment]
sukihanabi

**Comment by sukihanabi** [comment]
> Sorry one comment. I think the same fix should be applied to W2012 and W2012R2 versions.
> Thanks!

@dariomws306 Should this be added in the current repo or are you referring to a different repo?

**Comment by e0i** [comment]
> > Sorry one comment. I think the same fix should be applied to W2012 and W2012R2 versions.
> > Thanks!
> 
> @dariomws306 Should this be added in the current repo or are you referring to a different repo?

Thank should be this repository. Thanks.

**Comment by e0i** [comment]
@nenonix  Additional minor copy editing is needed for new PR content. Thanks.

**Comment by e0i** [comment]
@dariomws306 Please confirm that the content for the other two versions had been included appropriately. Thanks.

File: docset/winserver2012-ps/activedirectory/Set-ADAccountPassword.md [hunk 2]
```diff
@@ -34,10 +34,7 @@ Similarly, you can use Get-ADUser, Get-ADComputer or Get-ADServiceAccount, for s
 
 Note: Group MSAs cannot set password since they are changed at predetermined intervals.
 
-You must set the OldPassword and the NewPassword parameters to set the password unless you specify the Reset parameter.
-When you specify the Reset parameter, the password is set to the NewPassword value that you provide and the OldPassword parameter is not required.
-
-For AD LDS environments, the Partition parameter must be specified except in the following two conditions:
+For  Active Directory Lightweight Directory Services (AD LDS) environments, the Partition parameter must be specified except in the following two conditions:
```
Comment (nenonix) [inline docset/winserver2012-ps/activedirectory/Set-ADAccountPassword.md]:
```suggestion
For Active Directory Lightweight Directory Services (AD LDS) environments, the Partition parameter must be specified except in the following two instances:
```

File: docset/winserver2012r2-ps/addsadministration/Set-ADAccountPassword.md [hunk 3]
```diff
@@ -67,6 +64,7 @@ PS C:\>Set-ADAccountPassword -Identity elisada -OldPassword (ConvertTo-SecureStr
 ```
 
 This command sets the password of the user account with SamAccountName elisada to qwert@12345.
+Using -NewPassword with a value without providing an -OldPassword parameter value will also reset the password.
```
Comment (nenonix) [inline docset/winserver2012r2-ps/addsadministration/Set-ADAccountPassword.md]:
```suggestion
Using -NewPassword with a value, without providing an -OldPassword parameter value, will also reset the password.
```

File: docset/winserver2012-ps/activedirectory/Set-ADAccountPassword.md [hunk 4]
```diff
@@ -67,6 +64,7 @@ Description
 -----------
 
 Sets the password of the user account with SamAccountName: tmakovec to 'qwert@12345'.
+Using -NewPassword with a value without providing an -OldPassword parameter value will also reset the password.
```
Comment (nenonix) [inline docset/winserver2012-ps/activedirectory/Set-ADAccountPassword.md]:
```suggestion
Using -NewPassword with a value, without providing an -OldPassword parameter value, will also reset the password.
```

File: docset/windows/addsadministration/set-adaccountpassword.md [hunk 5]
```diff
@@ -68,6 +65,7 @@ PS C:\> Set-ADAccountPassword -Identity elisada -OldPassword (ConvertTo-SecureSt
 ```
 
 This command sets the password of the user account with SamAccountName elisada to qwert@12345.
+Using -NewPassword with a value without providing an -OldPassword parameter value will also reset the password.
```
Comment (nenonix) [inline docset/windows/addsadministration/set-adaccountpassword.md]:
```suggestion
Using -NewPassword with a value, without providing an -OldPassword parameter value, will also reset the password.
```

**Comment by dariomws306** [comment]
@e0i, looks good!

**Comment by e0i** [comment]
@AndreaBarr The PR can be merged. Thanks.

**Pull request merged**